<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="1" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-11T09:06:00 IST" name="Default suite" finished-at="2023-12-11T09:06:00 IST" duration-ms="43">
    <groups>
    </groups>
    <test started-at="2023-12-11T09:06:00 IST" name="Default test" finished-at="2023-12-11T09:06:00 IST" duration-ms="43">
      <class name="com.testng.Demo_3_DependsON">
        <test-method signature="createCustomerSBI()[pri:0, instance:com.testng.Demo_3_DependsON@201a4587]" started-at="2023-12-11T09:06:00 IST" name="createCustomerSBI" finished-at="2023-12-11T09:06:00 IST" duration-ms="7" status="FAIL">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 5 out of bounds for length 4]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 4
at com.testng.Demo_3_DependsON.createCustomerSBI(Demo_3_DependsON.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCustomerSBI -->
        <test-method signature="deleteCustomerSBI()[pri:-2, instance:com.testng.Demo_3_DependsON@201a4587]" started-at="2023-12-11T09:06:00 IST" name="deleteCustomerSBI" finished-at="2023-12-11T09:06:00 IST" depends-on-methods="com.testng.Demo_3_DependsON.createCustomerSBI" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.testng.Demo_3_DependsON.deleteCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587 depends on not successfully finished methods 
[com.testng.Demo_3_DependsON.createCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.testng.Demo_3_DependsON.deleteCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587 depends on not successfully finished methods 
[com.testng.Demo_3_DependsON.createCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteCustomerSBI -->
        <test-method signature="modifyCustomerSBI()[pri:1, instance:com.testng.Demo_3_DependsON@201a4587]" started-at="2023-12-11T09:06:00 IST" name="modifyCustomerSBI" finished-at="2023-12-11T09:06:00 IST" depends-on-methods="com.testng.Demo_3_DependsON.createCustomerSBI" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.testng.Demo_3_DependsON.modifyCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587 depends on not successfully finished methods 
[com.testng.Demo_3_DependsON.createCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.testng.Demo_3_DependsON.modifyCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587 depends on not successfully finished methods 
[com.testng.Demo_3_DependsON.createCustomerSBI() on instance com.testng.Demo_3_DependsON@201a4587]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifyCustomerSBI -->
      </class> <!-- com.testng.Demo_3_DependsON -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
